ts/features/idpay/initiative/details/components/InitiativeCardComponent.tsx

```diff
@@ -23,133 +24,157 @@ type Props = {
 };
 
 const styles = StyleSheet.create({
-  fullHeight: {
-    height: "100%"
-  },
   cardContainer: {
-    height: 230,
-    width: widthPercentageToDP(100)
-  },
-  card: {
-    position: "absolute",
-    alignSelf: "center",
-    width: widthPercentageToDP(100),
-    height: 230
+    backgroundColor: IOColors["blue-50"],
+    borderBottomEndRadius: 24,
+    borderBottomStartRadius: 24,
+    padding: 32,
+    paddingTop: 0,
+    flex: 1
   },
-  imageFull: {
-    zIndex: -1
+  badge: {
+    backgroundColor: IOColors.blue
   },
-  row: {
-    flexDirection: "row",
-    alignItems: "center",
-    textAlign: "center"
+  bonusLogoContainer: {
+    backgroundColor: IOColors.white,
+    height: 56,
+    width: 56,
+    borderRadius: 8
   },
-  spaced: {
-    justifyContent: "space-between"
+  topCardSection: {
+    flex: 2,
+    alignItems: "center"
   },
-  bottomInitiativeIcon: {
+  bottomCardSection: {
     flex: 1,
-    justifyContent: "flex-end"
-  },
-  paddedMainContent: {
-    zIndex: 1,
-    padding: 32,
-    width: "100%",
-    height: "100%",
-    position: "absolute"
-  },
-  amount: {
-    lineHeight: 32,
-    fontSize: 24,
-    color: IOColors.white,
-    ...makeFontStyleObject("Bold", undefined, "TitilliumWeb")
+    flexDirection: "row",
+    justifyContent: "space-evenly",
```

Comment:
```suggestion
    justifyContent: "center",
```

This fixes the spacing in the counters part. Note that you need to manually add a spacer now, see comment below.